README.md

```diff
@@ -9,128 +9,11 @@
 ## Installation
```

Comment:
Could you add summary & caution for 2 plugins to head of this document?

e.g.)
```
This repository contains 2 Fluentd plugins:

* in_windows_eventlog
* in_windows_eventlog2

The former one is obsolete, please don't use in newly deployment.
This document describes about the later one. If you want to know about the obsolete one, please see [in_windows_eventlog(old).md](in_windows_eventlog(old).md)
```

Reply:
I see! I added it!